src/target/adiv5.h

```diff
@@ -208,7 +208,8 @@
 #define JTAG_IDCODE_DESIGNER_OFFSET 1U
 #define JTAG_IDCODE_DESIGNER_MASK   (0x7ffU << JTAG_IDCODE_DESIGNER_OFFSET)
 
-#define JTAG_IDCODE_ARM_DPv0 UINT32_C(0x4ba00477)
+#define JTAG_IDCODE_ARM_DPv0    UINT32_C(0x0ba00476)
```

Comment:
This macro is not used after the changes, I think, so we can remove it as it's redundant